scalatest/src/main/scala/org/scalatest/enablers/Retrying.scala

```diff
@@ -59,7 +59,7 @@ object Retrying {
   private lazy val scheduler: ScheduledExecutorService = {
     val threadFactory = new ThreadFactory {
       val inner = Executors.defaultThreadFactory()
-      def newThread(runnable: Runnable) = {
+      def newThread(runnable: Runnable | Null) = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.